### DISCUSSION TITLE
What are the thresholds for each price category?

### DISCUSSION THREAD

**Discussion by Harro65** [amberelectric/public-api#57] [Q&A] [answered]
Does anyone know what the official logic is for each of the different prices categories?

I'm using the a number of utility_meter integrations with the tariffs, and some automations to set the current tarrif based on the current general amber price. 

At the moment im using the following ranges, but i know they are not matched to the app. 

Extremely low prices <10
Low prices >= 10 & < 20
Ok prices >= 20 & < 25
High prices >= 25 & < 40
Extremely high prices >= 40

**Comment by rob0101** [comment]
Everyone will have a different view of pricing value at different times.

E.g.   with TOU I could get charged 25c/kwh when the spot price is 5c and 15c when the spot price was 11c.
25c at peak (in SA) would be cheap and 15c at off-peak is expensive.

You should choose the method that suits you rather than Amber's classifications that try to generalise over many customers.

**Comment by madpilot** [comment, marked as answer]
I was waiting for someone to ask this.

The answer is complicated, and depends on your network tariff.

If you are on an anytime tariff, the thresholds are:

Low: spot price < 8c
Average: spot price < 14c
High: spot price < 300c
Spike: spot price >= 300c

For a time of use tariff, we take the average price of all the tariff times (peak/shoulder/off peak) them subtract that from the perKWH price, then apply the above logic to that number.
